### PR TITLE
feat/AUT-964/ e2e Item Authoring, Assign correct answer

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-964

Covered by e2e testing page item Authoring. Assigned correct answers (where possible)

**How to test:**
- switch tao to branch https://github.com/oat-sa/tao-core/pull/3147
- run cypress from tao/views 
- check test `taoQtiItem/views/cypress/tests/item-authoring-set-response.spec.js`

### CODE REVIEW

**Review by Silvia-loza — APPROVED** [review]
Great idea to create a separate file as item-authoring-spec.js was becoming too long.
All tests pass consistently.
- [ ] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: views/cypress/tests/item-authoring-set-response.spec.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                    className,
+                    selectors.deleteClassUrl,
+                    selectors.resourceRelations,
+                    false,
+                    true
+                );
+            }
+        });
+    });
+    after(() => {
```
Comment (adriloma) [inline views/cypress/tests/item-authoring-set-response.spec.js:85]: Maybe it is helpful to comment what `after` hook does.

File: views/cypress/tests/item-authoring-set-response.spec.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                className,
+                selectors.editClassLabelUrl,
+                selectors.treeRenderUrl,
+                selectors.addSubClassUrl
+            );
+            cy.addNode(selectors.itemForm, selectors.addItem);
+            cy.renameSelectedNode(selectors.itemForm, selectors.editItemUrl, itemName);
+
+            // cy.selectNode(selectors.root, selectors.itemClassForm, className);
+            // cy.contains('a', itemName).click();
```
Comment (adriloma) [inline views/cypress/tests/item-authoring-set-response.spec.js:123]: Is this commented code intentional?

**Review by adriloma — APPROVED** [review]
All good, just two minor comments 👍 

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following conventional commits
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Pull request merged**